packages/nodes-base/nodes/Onfleet/descriptions/TeamDescription.ts

```diff
@@ -83,6 +93,169 @@ const enableSelfAssignmentField = {
 	description: 'This toggles Team Self-Assignment that allows Drivers to Self Assign Tasks that are in the Team unassigned container.',
 } as INodeProperties;
 
+const maxTasksPerRouteField = {
+	displayName: 'Max number of tasks per route',
+	name: 'maxTasksPerRoute',
+	type: 'number',
+	default: 100,
+	typeOptions: {
+		maxValue: 200,
+		minValue: 1,
+	},
+	description: 'Total number of tasks allowed on a route.',
+}as INodeProperties;
+
+const taskTimeWindowField = {
+	displayName: 'Task time window (JSON)',
+	name: 'taskTimeWindow',
+	type: 'json',
+	default: '{}',
+	description: 'This is the time window of tasks to include in the optimization. Param must be an array of length 2 in unix time in seconds precision, [start, end]',
+} as INodeProperties;
+
+const scheduleTimeWindowField = {
+	displayName: 'Schedule time window (JSON)',
+	name: 'scheduleTimeWindow',
+	type: 'json',
+	default: '{}',
+	description: 'This is the Driver\'s scheduled time window. Param must be an array of length 2 in unix time in seconds precision, [start, end]',
+} as INodeProperties;
+
+const serviceTImeField = {
```

Comment:
`serviceTimeField`

Reply:
Here the default service time it's 2 since this it's the auto dispatch endpoint